spec/unit/knife/core/bootstrap_context_spec.rb

```diff
@@ -20,6 +20,10 @@
 require "chef/knife/core/bootstrap_context"
 
 describe Chef::Knife::Core::BootstrapContext do
+  before do
+    Chef::Config[:fips] = false
```

Comment:
@danielsdeleo  Ok, so it's coming back to me: This was required to get tests to pass in both pipelines. The environment CHEF_FIPS is set to 1 for the chef-fips pipeline, which causes the default value to change to true for that pipeline.